index.js

```diff
@@ -133,6 +134,9 @@ SassCompiler.prototype._rubyCompile = function(source, callback) {
       cmd.push(hasComments ? '--line-comments' : '--debug-info');
     }
 
+    if (this.config.precision) {
```

Comment:
Because I'm a dummy and my fingers are trained for curly braces in JavaScript. Thank you, Douglas Crockford! Changing now...